src/scancode/cli.py

```diff
@@ -458,11 +465,19 @@ def scancode(ctx,
             full_root=full_root,
             ignore=user_ignore)
 
+        for option, process in plugincode.post_scan.get_post_scan_plugins().items():
+            if kwargs[option.replace('-', '_')]:
+                options['--' + option] = True
+                if not quiet:
+                    echo_stderr('Running post-scan plugin: %(option)s...' % locals(), fg='green')
+                results = process(scanners, results, options, input)
```

Comment:
`process` is called for a `post-scan` plugin only if user supplied its corresponding option,
`scanners` are called in the `scan` function [here](https://github.com/nexB/scancode-toolkit/blob/7f2fb29d21e42b394b58ab1680bd0bccf5a2f6f3/src/scancode/cli.py#L455)
where as post-scan plugins are called [here](https://github.com/nexB/scancode-toolkit/blob/7f2fb29d21e42b394b58ab1680bd0bccf5a2f6f3/src/scancode/cli.py#L473)